packages/api/models/grantCycle.js

```diff
@@ -57,8 +57,8 @@ module.exports = (sequelize, DataTypes) => {
           if (instance.isActive) {
             try {
               const activeGrant = await GrantCycle.findOne({ where: { isActive: true } });
-              if (activeGrant) {
-                throw new ValidationError('Active grant already exists: ', activeGrant.name);
+              if (activeGrant && instance.id !== activeGrant.id) {
+                throw new ValidationError(`Active grant already exists: ${activeGrant.name}`);
```

Comment:
I went ahead and merged this, but thinking about it, it might be good to add a test or two on the controller to handle updating active grants.